Mathlib/Analysis/Calculus/Deriv/Comp.lean

```diff
@@ -125,6 +125,8 @@ theorem HasDerivAtFilter.comp_hasFDerivAtFilter {f : E → 𝕜'} {f' : E →L[
     (hL : Tendsto f L'' L') : HasFDerivAtFilter (h₂ ∘ f) (h₂' • f') x L'' := by
   convert (hh₂.restrictScalars 𝕜).comp x hf hL
   ext x
+  dsimp
+  rw [Pi.smul_apply]
```

Comment:
Still have `simp only [Pi.smul_apply]` failures but those seem to be part of a larger problem as seen with the recent core change.